ios/NativeFidelBridge.m

```diff
@@ -8,4 +8,6 @@ @interface RCT_EXTERN_MODULE(NativeFidelBridge, NSObject)
 
 RCT_EXTERN_METHOD(verifyCard:(NSDictionary *)cardVerificationConfiguration)
 
+RCT_EXTERN_METHOD(identifyMetricsDataSource:(NSString *)name version:(NSString *)version)
```

Comment:
Can you explain to me what is the goal of this file, and if we need to do the same for Android?

Reply:
Yes, of course. This file creates the interface that enables Javascript to call native functions (in this case it’s the `identifyMetricsDataSource` function). The macro on this line defines an "external method” for Javascript, based on the `identifyMetricsDataSource` iOS native function.

Android has to align with this external function definition and it does, by declaring it [here](https://github.com/FidelLimited/rn-sdk/pull/129/files#diff-c8baa503d2d6ed8ebec03fbb697976a20ed30c13b6884d9b784bbdc55cdb8ba5R104).